test/Infrastructure/LeanCode.SendGrid.Tests/SendGridRazorClientTests.cs

```diff
@@ -59,19 +59,28 @@ public SendGridRazorClientTests()
     }
 
     [SendGridFact]
-    public async Task Sends_email_correctly()
+    public async Task Sends_localized_email_correctly()
     {
         var msg = new SendGridLocalizedRazorMessage("pl")
             .WithSender(EmailFrom, "LeanCode Tester")
             .WithRecipient(EmailTo)
             .WithSubject("email.subject.test")
             .WithPlainTextContent(new EmailTextVM())
             .WithHtmlContent(new EmailHtmlVM())
-            .WithAttachment(
-                Convert.ToBase64String(Encoding.UTF8.GetBytes("Attachment content.")),
-                "Attachment.txt",
-                "text/plain"
-            )
+            .WithAttachment(Convert.ToBase64String("Attachment content."u8.ToArray()), "Attachment.txt", "text/plain")
```

Comment:
While you're at it, we could switch to the overload that accepts `ROS<byte>` directly.

```suggestion
            .WithAttachment(Convert.ToBase64String("Attachment content."u8), "Attachment.txt", "text/plain")
```